src/VirtualClient/VirtualClient.Actions/SuperBenchmark/SuperBenchmarkExecutor.cs

```diff
@@ -168,10 +170,25 @@ protected override async Task InitializeAsync(EventContext telemetryContext, Can
                         file,
                         this.Combine(this.SuperBenchmarkDirectory, Path.GetFileName(file)),
                         true);
+
+                    await this.ExecuteCommandAsync("dos2unix", $"{Path.GetFileName(file)}", this.SuperBenchmarkDirectory, telemetryContext, cancellationToken, runElevated: true);
```

Comment:
Great find! Let's just change the line ending in the vc package itself instead of using a command